pycbc/strain/strain.py

```diff
@@ -284,12 +319,16 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
     if opt.zpk_z and opt.zpk_p and opt.zpk_k:
```

Comment:
Similar to early sections, the conditioning is largely duplciated. Can this not be turned into a loop / modular function, so the lines aren't duplicated?